test/fw/ptl/utils/pbs_logutils.py

```diff
@@ -256,7 +256,7 @@ def convert_date_time(cls, dt=None, fmt=None):
             # Get epoch-timestamp assuming local timezone
             tm = t.timestamp()
         except ValueError:
-            cls.logger.info("could not convert date time: " + str(dt))
+            cls.logger.debug("could not convert date time: " + str(dt))
```

Comment:
I made the change because the amount of output from that line of code was substantial.  Some of the messages logged by the server contain newlines, and this code logs a message for each of those newlines every time a `log_match()` is performed.  The logging level was debug until recently, and having it set to info was making it significantly harder to read logging output related to the test.  I can change it back, but it seemed like it was there for the purposes of debugging PTL and thus the debug level was more appropriate.